examples/cosmos/ics20/denomTrace.qnt

```diff
@@ -173,6 +188,87 @@ module denomTraceTest {
   }))
 }
 
+module denomTraceSimulation {
```

Comment:
I think it would help to include a high-level intro comment of what's achieved by this module.

Since we don't have additional documentation, maybe also include the commands used to run it from the PR description?